pages/task/read/routers/confirm.js

```diff
@@ -11,7 +11,12 @@ module.exports = () => {
 
   app.use(form({
     locals: (req, res, next) => {
+      const formValues = req.session.form[`${req.task.id}-decision`].values;
+      const decision = formValues.decision;
+
       res.locals.static.task = req.task;
+      res.locals.static.decision = decision;
+      res.locals.static.reason = formValues[`${decision}-reason`];
```

Comment:
Does this need to be sent down in the locals or can it be deduced on the client?

I'd like to try to avoid dumping too many things onto locals if they can be calculated from existing values. The script logic is cacheable, but adding locals is not cached because they're sent inline in the page.